dcpy/connectors/edm/recipes.py

```diff
@@ -192,6 +192,14 @@ def to_row(self) -> dict:
         }
 
 
+def archive(config: Config, file_path: Path):
+    ### this could either take a config object and path to a file, and archive them both in a folder together,
+    ### or take path to folder that takes dumped config as well as file output and loads whole folder
+    ### likely the latter in case of multiple output formats. Not necessarily something we want to be doing long term
+    ### but for flexibility for now it might make sense
+    raise NotImplemented
```

Comment:
Is this function responsible for creating desired output formats (i.e. csv, pgdump) or would it be a separate fn? If it will happen here, then the latter option in your comment shouldn't be a concern

Reply:
I would say it should happen prior to this, but open for debate

Reply:
I noticed that we are archiving raw data in `ingest.py`. What do you think about moving this function to `ingest.py`?

Reply:
Maybe. My gut is have it here, just because there are a few things going on in extract
- grab raw data
- push raw data to s3 (edm-recipes)
- transform to parquet
- process
- push to s3 (edm-recipes)

I think the interaction with s3 portions should live within connectors.edm.recipes, and not use our s3 utils directly. We're definitely blurring the line a little bit here, and there's an argument to unify a bit more code. But to me, we're keeping `extract` logic as the logic around pulling data, archiving it, and getting it ready for our pipelines, not dealing with the actual storage/representation of datasets that we've archived. So that logic should stay in recipes

Reply:
With the rationale being that our extract code doesn't need to know about the details of where we're storing things, our edm recipes connector logic can take care of that

Reply:
makes sense